torch/_dynamo/trace_rules.py

```diff
@@ -3021,16 +3021,35 @@ def _polyfilled_function_ids() -> Set[int]:
 
 @FunctionIdSet
 def _numpy_function_ids() -> Dict[int, str]:
+    unsupported_funcs = {
```

Comment:
You might not need this set - if we actually encounter an unsupported numpy function, I believe we would just graph break when attempting to trace the call in dynamo. You can try removing this set and seeing if the tests pass.

Reply:
I have update the PR.
I did some quick local tests. They passed.
Thanks!